### ISSUE TITLE
does not run or install from source

### ISSUE THREAD

**Issue by carandraug** [macronucleus/chromagnon#2]
The available for releases are for windows and mac only.  I am trying to run it on Linux but found some issues.  I am unsure how it is supposed to be handled, so I tried all the ones I know.

The source releases are not python packages (but looks like they should be since `setup.py` does have an install target and define `main.py` as a package script):

```
$ wget https://github.com/macronucleus/Chromagnon/archive/v0.5.tar.gz
$ pip install v0.5.tar.gz 
Unpacking ./v0.5.tar.gz
  Running setup.py (path:/tmp/pip-FB63kz-build/setup.py) egg_info for package from file:///home/carandraug/builds/v0.5.tar.gz
    Traceback (most recent call last):
      File "<string>", line 17, in <module>
      File "/tmp/pip-FB63kz-build/setup.py", line 190, in <module>
        version=version,
    NameError: name 'version' is not defined
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):

  File "<string>", line 17, in <module>

  File "/tmp/pip-FB63kz-build/setup.py", line 190, in <module>

    version=version,

NameError: name 'version' is not defined

----------------------------------------
Cleaning up...
Command python setup.py egg_info failed with error code 1 in /tmp/pip-FB63kz-build
Storing debug log for failure in /home/carandraug/.pip/pip.log
```

The released sources are not an application ready to run either:

```
$ tar -xzf v0.5.tar.gz 
$ cd Chromagnon-0.5/
$ python main.py 
this is not frozen
Traceback (most recent call last):
  File "main.py", line 55, in <module>
    from PriCommon import guiFuncs as G, bioformatsIO, commonfuncs as C, listbox
  File "/home/carandraug/builds/Chromagnon-0.5/PriCommon/bioformatsIO.py", line 73, in <module>
    from mybioformats import omexml as ome
ImportError: No module named mybioformats
```

Running from development sources also fails:

```
$ git clone https://github.com/macronucleus/Chromagnon.git
$ cd Chromagnon/
$ python main.py 
carandraug@mprocessor1:~/dev/Chromagnon$ python main.py 
Traceback (most recent call last):
  File "main.py", line 46, in <module>
    from PriCommon import guiFuncs as G, bioformatsIO, commonfuncs as C, listbox
ImportError: cannot import name listbox
```

Trying to install from development sources also fails:

```
$ ./setup.py install
Traceback (most recent call last):
  File "./setup.py", line 190, in <module>
    version=version,
NameError: name 'version' is not defined
```

I added a dummy `version` variable to the top of `setup.py` but that still caused issues:

```
$ ./setup.py install
running install
running bdist_egg
running egg_info
creating Chromagnon.egg-info
writing requirements to Chromagnon.egg-info/requires.txt
writing Chromagnon.egg-info/PKG-INFO
writing top-level names to Chromagnon.egg-info/top_level.txt
writing dependency_links to Chromagnon.egg-info/dependency_links.txt
writing manifest file 'Chromagnon.egg-info/SOURCES.txt'
error: package directory 'chromagnon' does not exist
$ ./setup.py sdist
running sdist
running egg_info
writing requirements to Chromagnon.egg-info/requires.txt
writing Chromagnon.egg-info/PKG-INFO
writing top-level names to Chromagnon.egg-info/top_level.txt
writing dependency_links to Chromagnon.egg-info/dependency_links.txt
error: package directory 'chromagnon' does not exist
```

**Comment by carandraug** [comment]
In one last attempt, I tried to checkout a specific tag, but that also didn't work:

```
$ git checkout v0.5 
$ python setup.py install
running install
running bdist_egg
running egg_info
writing requirements to Chromagnon.egg-info/requires.txt
writing Chromagnon.egg-info/PKG-INFO
writing top-level names to Chromagnon.egg-info/top_level.txt
writing dependency_links to Chromagnon.egg-info/dependency_links.txt
error: package directory 'chromagnon' does not exist
```

**Comment by carandraug** [comment]
Fixed with 07b0be56c88a644b6b9ec6523b6c6bb2dce00fa0
